test/models/work_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { Work.new }
+  let(:lemonade) { works(:lemonade)}
+  let(:hellodolly) { works(:hellodolly)}
+  let(:vote_a) { votes(:vote_a)}
+  let(:vote_b) { votes(:vote_b)}
+  let(:vote_c) { votes(:vote_c)}
+  let(:user_a) { users(:user_a)}
+  let(:user_b) { votes(:user_b)}
+
+  describe "validations" do
+    it "is not valid without a title and author" do
+      work.valid?.must_equal false
```

Comment:
You would need something to verify that the title & author both caused the validations to fail